controller/src/test/java/org/jboss/as/controller/persistence/PersistanceResourceTestCase.java

```diff
@@ -370,6 +371,17 @@ public void testRelativePathFromOutsideConfigDirectory() throws Exception {
         }
     }
 
+    @Test
+    public void testSymbolicLinkForConfigurationFiles() throws Exception {
+        Path file = Paths.get(createFile(externalDir, "standard.xml", "test").toString());
+        Path symbolicLink = standardDir.toPath().resolve("standard.xml");
+        if(Files.exists(symbolicLink)){
+            Files.delete(symbolicLink);
+        }
+        Files.createSymbolicLink(symbolicLink, file);
+        ConfigurationFile configurationFile = new ConfigurationFile(standardDir, "standard.xml", file.toString(), false);
```

Comment:
You are using here wrong arguments:
* The third argument is expecting the name of the file provided by the user relative to the configuration dir, you are passing `file.toString()`, which is incorrect. Leave it as null or just "standard.xml"
* The fourth argument is `false`, which means the changes are not persisted in the "standard.xml" file by enabling `InteractionPolicy.READ_ONLY`. That will make the code used by ConfigurationFile return before testing changes giving a false negative result. You have to use `true`instead